### DISCUSSION TITLE
Error when connecting to an ngx-socketio client

### DISCUSSION THREAD

**Discussion by Arti-23** [socketio/socket.io#4353] [Q&A]
an error occurs when trying to connect from an ngx-socketIo client, this is the service

![image](https://user-images.githubusercontent.com/78559244/165323154-093f8276-7878-48be-9d1c-4bd881659bc9.png)

this is the client

![image](https://user-images.githubusercontent.com/78559244/165324498-fa48483b-afc7-4d61-b1b7-0ebfcb5886e3.png)

and that is the error

![image](https://user-images.githubusercontent.com/78559244/165324002-eba97954-97cd-49f6-b422-63963ce7195c.png)

but when i use 'new webSocket' manually works succcefully

![image](https://user-images.githubusercontent.com/78559244/165324217-b6b15de3-143f-4f36-817c-4b5f2b6587d9.png)

![image](https://user-images.githubusercontent.com/78559244/165324258-51d67f1b-9fd8-4d80-b5d3-5bc04172f17c.png)

**Comment by darrachequesne** [comment]
Hi! It seems you are trying to reach a plain WebSocket server with a Socket.IO client, which is not possible.

Reference:

- https://socket.io/docs/v4/#what-socketio-is-not
- https://socket.io/docs/v4/troubleshooting-connection-issues/
